plugins/thunderbolt/fu-self-test.c

```diff
@@ -902,6 +902,12 @@ test_set_up (ThunderboltTest *tt, gconstpointer params)
 		g_assert_nonnull (tt->tree);
 	}
 
+	if (!umockdev_in_mock_environment ()) {
+		g_warning ("Need to run within umockdev wrapper (umockdev-wrapper %s)!",
```

Comment:
Maybe `g_critical` or `g_error`?

Reply:
Yeah g_critical makes sense, i'll adjust and pull in.